.travis.yml

```diff
@@ -1,10 +1,10 @@
-sudo: required
```

Comment:
what's this change for?

Reply:
<img width="603" alt="Screen Shot 2020-02-26 at 9 36 30 AM" src="https://user-images.githubusercontent.com/3122764/75366181-91e2c100-587b-11ea-8521-e178e7eb83ab.png">